azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/service/sdk/CallingSDK.kt

```diff
@@ -3,6 +3,7 @@
 
 package com.azure.android.communication.ui.calling.service.sdk
 
+/*  <CALL_START_TIME> */
```

Comment:
Same, remove any flags in import sections. If CALL_START_TIME needs disabling, we can fix the breakages when they happen by adjusting other flags.